Makefile

```diff
@@ -19,7 +19,8 @@ MAILBOXD_MANAGER_PIDFILE ?= /opt/zimbra/log/zmmailboxd_manager.pid
 MAILBOXD_MANAGER_DEPRECATED_PIDFILE ?= /opt/zimbra/log/zmmailboxd.pid
 MAILBOXD_JAVA_PIDFILE ?= /opt/zimbra/log/zmmailboxd_java.pid
 MAILBOXD_CWD ?= /opt/zimbra/log
-MAILBOXD_HOME ?= /opt/zimbra/mailboxd
+JETTY_BASE ?= /opt/zimbra/mailboxd
+JETTY_HOME ?= /opt/jetty9
```

Comment:
Where does upstream jetty install?  I assume we will still be packaging a Zimbra specific version of Jetty?  If so, it would be good to make it extremely unlikely that we clash with other Jetty providers.